src/onboarding/onboarding-core-config.ts

```diff
@@ -248,6 +292,14 @@ class OnboardingCoreConfig extends LitElement {
 
   private async _save(ev) {
     ev.preventDefault();
+    if (this._autoDetect) {
+      await this._detect();
+      if (!this._autoDetect) {
+        // We set _autoDetect to false in _detect if it failed
```

Comment:
Then you should disable the controls while saving/detecting, but I think it would be better to have `this._detect()` return a bool.